iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/AbstractTransformerXml.java

```diff
@@ -46,7 +46,7 @@ protected static void writeTapChangerStep(TapChangerStep<?> tcs, XMLStreamWriter
     protected static void writeTapChanger(TapChanger<?, ?> tc, XMLStreamWriter writer) throws XMLStreamException {
         writer.writeAttribute(ATTR_LOW_TAP_POSITION, Integer.toString(tc.getLowTapPosition()));
         writer.writeAttribute(ATTR_TAP_POSITION, Integer.toString(tc.getTapPosition()));
-        XmlUtil.writeDouble(TARGET_DEADBAND, tc.getTargetDeadband(), writer);
+        XmlUtil.writeOptionalDouble(TARGET_DEADBAND, tc.getTargetDeadband(), 0, writer);
```

Comment:
I think you have to change this since I merged #1054 ?

Reply:
Don't think so, using `writeDouble` instead of `writeOptionalDouble` was a mistake I think...

Reply:
Are you sure we don't want to serialize `targetDeadband` if the value is `0`? Is it really a good idea?

Reply:
In 1.0, `targetDeadband` was optionally serialized if not defined (i.e. at default value which was considered during calculations as if it was equal to 0). We can make it systematically serialized if it is less confusing?

Reply:
If I understand well, the `targetDeadband` attribute doesn't exist in fist versions of powsybl. We add this attribute recently (end of June 2019). You add this attribute as optional in the XSD to manage the compatibility.

Looking to the master, the attribute is read from the file but never write, right? The allowed values are `NaN` or a value greater than `0`. How do you distinguish that the value was NaN or 0 if you don't export it when it's equal to 0?

Another thing you can do is to convert NaN values to 0. In that case you have to fix the adder.

Reply:
Changed the behavior of `targetDeadband`: it is by default undefined (`Double.NaN`) and is required when the equipment is regulating. For backward compatibility, `targetDeadband` is not serialized in 1.0 when it equals `0` and **in 1.0**, if it is absent when the equipment is regulating, it is considered as equal to 0.